app/session.go

```diff
@@ -93,6 +93,28 @@ func (a *App) GetSessions(userId string) ([]*model.Session, *model.AppError) {
 	return a.Srv.Store.Session().GetSessions(userId)
 }
 
+func (a *App) UpdateSessionsIsGuest(userId string, isGuest bool) {
+	sessions, err := a.Srv.Store.Session().GetSessions(userId)
+	if err != nil {
+		mlog.Error(fmt.Sprintf("Unable to get user sessions: userId=%s err=%s", userId, err.Error()))
+	}
+
+	for _, session := range sessions {
+		if isGuest {
+			mlog.Error("UPDATING SESSIONS TO GUEST")
+			session.AddProp(model.SESSION_PROP_IS_GUEST, "true")
+		} else {
+			mlog.Error("UPDATING SESSIONS TO NOT GUEST")
+			session.AddProp(model.SESSION_PROP_IS_GUEST, "false")
+		}
+		_, err := a.Srv.Store.Session().Save(session)
+		if err != nil {
+			mlog.Error(fmt.Sprintf("Unable to update isGuest session: %s", err.Error()))
```

Comment:
after the error, shouldn't there be a 'continue' here?

Reply:
Fixed, actually I added unit tests to verify the correct behavior, and now is correct.